### PR TITLE
Update to Pac CLI v1.30.3 and update PacTypes `AuthProfileListing` to new schema coming from pac CLI

### PR DESCRIPTION
**Pull request by joem-msft**
The auth profiles no longer have a 'Resource' property. Instead, there is a new property for 'ActiveOrganization' which is set iif the profile has an active organization configured for it. This property is an object which has the 'environmentUrl' and 'friendlyName' properties available.

### CODE REVIEW

File: src/client/pac/PacTypes.ts [hunk 1]
```diff
@@ -75,3 +66,5 @@ export type ActiveOrgOutput = {
     UserId : string,
     EnvironmentId: string,
 }
+
+export type PacActiveOrgListOutput = PacOutputWithResultList<ActiveOrgOutput>;
```
Comment (tehcrashxor) [inline src/client/pac/PacTypes.ts:70]: The Result object output here from `pac org who` is a single object, not a collection.

File: src/client/test/unit/PacWrapper.test.ts [hunk 2]
```diff
@@ -33,13 +33,14 @@ class MockPacInterop implements IPacInterop {
 describe('PacWrapper', () => {
     it('AuthList parses correctly', async () => {
         const interop = new MockPacInterop();
-        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"Resource\":\"https://contoso.example.com\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}]}";
+        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"ActiveOrganization\":{\"EnvironmentUrl\":\"https://contoso-mock.crmtest.dynamics.com\",\"FriendlyName\":\"\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}]}";
```
Comment (tehcrashxor) [inline src/client/test/unit/PacWrapper.test.ts]: Missing closing `}` for the `ActiveOrganization`

File: src/client/test/unit/PacWrapper.test.ts [hunk 3]
```diff
@@ -33,13 +33,14 @@ class MockPacInterop implements IPacInterop {
 describe('PacWrapper', () => {
     it('AuthList parses correctly', async () => {
         const interop = new MockPacInterop();
-        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"Resource\":\"https://contoso.example.com\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}]}";
+        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"ActiveOrganization\":{\"EnvironmentUrl\":\"https://contoso-mock.crmtest.dynamics.com\",\"FriendlyName\":\"\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}}]}";
```
Comment (tehcrashxor) [inline src/client/test/unit/PacWrapper.test.ts:36]: Looks like the new closing `}` is in the wrong place - should be after the ActiveOrganization's object

File: README.md [hunk 4]
```diff
@@ -20,6 +20,9 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 
 ## Release Notes
 
+2.0.25:
```
Comment (tehcrashxor) [inline README.md:23]: Should probably set this version number to `2.0.27`
There will be at least two "releases" - the preview one and the standard release, which can't share version numbers between the two types

**Review by tehcrashxor — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (joem-msft) [reply README.md:23]: will use the placeholder NEXTRELEASE for this and allow it to be specified during our regular release process.

**Pull request merged**